environment/src/test/java/com/sequenceiq/environment/credential/service/ServiceProviderCredentialAdapterTest.java

```diff
@@ -96,7 +96,7 @@ class ServiceProviderCredentialAdapterTest {
 
     @BeforeEach
     void setUp() throws InterruptedException {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
Just a side note:
Ideally we do not need any manual init/open of mocks as `@InjectMocks` annotation has been added to the tested class